### DISCUSSION TITLE
Fan card puhlees

### DISCUSSION THREAD

**Discussion by mouth4war** [home-assistant/frontend#8892] [Dashboard]
So a fan card PR https://github.com/home-assistant/frontend/pull/7803 was submitted 5 months ago and got shot down since it used round sliders like the light card still does. Can it be merged? 

Fan platform now has a speed slider as well so it's more ready to be treated like the light card than before.
Whenever the light card is changed, the fan card can be  updated too.

**Comment by Wyren** [comment]
I just openend an other request for this. Why didn't they add this yet. That slider is still there for crying out loud...
